### ISSUE TITLE
Empty View when in Main Tab

### ISSUE THREAD

**Issue by negue** [joshuatazrein/obsidian-time-ruler#55]
**Describe the bug**
Very often, when the Time-Ruler is in the Main Tabs, the full view is empty. 

Maybe this is also the reason the Issue reported in #53

**To Reproduce**
1. Open the ruler
2. Move the Tab to the Main Tabs
3. Close Obsidian
4. Open it again And "often" (sometimes it works, awesome bugs..) it breaks into an empty view

**Obsidian Version:** 
- [ ] Mobile
- [x] Desktop

**Additional context**

OS: Mac 13.4.1 (c) (22F770820d)
Plugin Version: 1.3.1

Probably a timing issue, haven't checked the code yet

Even reloading Obsidian its not showing up afterwards,  its strange 😬 

Console Errors:

```
// (3x times the following)

Uncaught TypeError: Failed to execute 'observe' on 'ResizeObserver': parameter 1 is not of type 'Element'.
    at eval (plugin:time-ruler:70148:14)
    at commitHookEffectListMount (plugin:time-ruler:31778:34)
    at commitPassiveMountOnFiber (plugin:time-ruler:33031:17)
    at commitPassiveMountEffects_complete (plugin:time-ruler:32999:17)
    at commitPassiveMountEffects_begin (plugin:time-ruler:32989:15)
    at commitPassiveMountEffects (plugin:time-ruler:32979:11)
    at flushPassiveEffectsImpl (plugin:time-ruler:34360:11)
    at flushPassiveEffects (plugin:time-ruler:34317:22)
    at performSyncWorkOnRoot (plugin:time-ruler:33738:11)
    at flushSyncCallbacks (plugin:time-ruler:24009:30)
    at commitRootImpl (plugin:time-ruler:34302:11)
    at commitRoot (plugin:time-ruler:34147:13)
    at finishConcurrentRender (plugin:time-ruler:33675:15)
    at performConcurrentWorkOnRoot (plugin:time-ruler:33588:15)
    at workLoop (plugin:time-ruler:15071:42)
    at flushWork (plugin:time-ruler:15050:22)
    at Immediate.performWorkUntilDeadline (plugin:time-ruler:15258:29)
    at process.processImmediate (node:internal/timers:476:21)

-------------------

plugin:time-ruler:28910 The above error occurred in the <App2> component:

    at App2 (plugin:time-ruler:69977:17)

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://reactjs.org/link/error-boundaries to learn more about error boundaries.

-------------------

plugin:time-ruler:68344 TypeError: Cannot destructure property 'folder' of '(intermediate value)' as it is null.
    at getDailyNoteInfo (plugin:time-ruler:68330:11)
    at async setupStore (plugin:time-ruler:69979:27)
```

**Comment by negue** [comment]
Just saw the there is already an update to 1.3.3 (not in Obsidian avaiable?) - will try that version using "BRAT" - and report if the issue still exists

**Comment by negue** [comment]
I can confirm the errors are still existing in the 1.3.3

**Comment by joshuatazrein** [comment]
I hope this is fixed in upcoming 1.4.0, let's see if it is.
